db/migrate/20211204093104_create_batches.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBatches < ActiveRecord::Migration[6.1]
+  def change
+    create_table :batches do |t|
+      t.references :branch
+      t.references :coaching
+      t.string :coach_name
+      t.integer :players_limit
+      t.integer :players_count
+      t.integer :start_at_in_HH
```

Comment:
what is HH?
incorrect naming convention. dont upcase

Reply:
Naming fixed.